CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+</p>
+
+On mobile, tooltips are not activatable in Webkit.
+
+### Longer tooltips across browsers
+
+Below is an overview of how each major browser engine handles longer tootlip text,
+in no particular order.
+
+#### Chromium
+
+In Chromium, built-in tooltips add elipses to the tooltip text if the text
+is greater than or equal to 1025 characters. The tooltip is also allowed to
+escape the bounds of the window. For example:
+
+<p align="center">
+  <image src="images/out-of-bounds-chromium-windows.png">
+</p>
+
+#### Gecko
+
+In Gecko, I was *not* able to find a limit for built-in tooltip length for
+which an elipses was added, like it does in Chromium. However, tooltips
+are also able to escape the bounds of the window in Gecko, but the tooltip
+width stays more constrained than in Chromium. For example:
+
+<p align="center">
+  <image src="images/out-of-bounds-firefox-windows.png">
+</p>
+
+#### Webkit
+
+In Webkit, I was *not* able to find a limit for built-in tooltip length for
+which an elipses was added, like it does in Chromium. However, tooltips
+are also able to escape the bounds of the window in Webkit. In this case,
+the tooltip appears to have uneven padding. For example:
+
+<p align="center">
+  <image src="images/out-of-bounds-webkit-mac.png">
+</p>
+
+_________________________
+
+It is clear that the tooltips provided by the top browser engines are
+inconsistent in styling between each other and across OS, and there are
+divergences in positioning and some behaviors with longer tooltip text.
+
+This inconsistency and lack of customizability leads authors to find
+alternatives that often require more work and maintenance.
+
+## The `::tooltip` pseudo element
+
+Many have proposed a pseudo for styling browser tooltips over the years,
+with the [first proposal](
+https://lists.w3.org/Archives/Public/www-style/2000Apr/0014.html)
+originating around April 2000, and the most [recent proposal](
+https://github.com/w3c/csswg-drafts/issues/8930#issuecomment-1581848840)
+from Lea Verou in 2023. 
+
+The proposal in this document takes close inspiration from these, by
+introducing a new pseudo element, called `::tooltip`, that can be
+used by authors to style the browser tooltip that is displayed when
+the `title` attribute is set on an HTML element.
+
+The `::tooltip` pseudo element will have default UA styles that will
+be required to be applied when an author adjusts tooltip styles
+within a `::tooltip` pseudo element selector.
+
+The idea behind this proposal is that authors would be able to
+create simple text-based tooltips, and adjust the styles of color
+properties, shadowing, border properties, font properties, etc
+to better match the look and feel of their site, leaving
+positioning, input handling, and accessibility up to the browser.
+
+### How to trigger `::tooltip` default styles?
+
+Currently, browser default tooltips are triggered by the `title`
+attribute being set, or `<title>` being set within an SVG element.
```

Comment:
"or `<title>` being set within an SVG element." Maybe phrase this as "or the presence of a `<title>` child node for SVG elements."